docs/configure/styling-and-css.md

```diff
@@ -2,20 +2,32 @@
 title: 'Styling and CSS'
 ---
 
-There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually, it is best to try and replicate what your application does with styling in Storybook’s configuration.
+There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually, it is best to try and replicate what your application does with styling in Storybook’s configuration. To make this easier, we recommend using [`@storybook/addon-styling`](https://github.com/storybookjs/addon-styling).
 
-### CSS-in-JS
+## Importing CSS files
 
-CSS-in-JS libraries are designed to use basic JavaScript, and they often work in Storybook without any extra configuration. Some libraries expect components to render in a specific rendering “context” (for example, to provide themes), and you may need to add a [global decorator](../writing-stories/decorators.md#global-decorators) to supply it.
+Storybook is pre-configured to recognize imports for css files. To add global CSS for all of your stories, import it in [`.storybook/preview.js`](./overview.md#configure-story-rendering).
 
-### Importing CSS files
+```js
+// .storybook/preview.js
 
-If your component files import their CSS, Storybook's webpack configuration will work out of the box. The noticeable exception to this is if you're using a CSS precompiler. In this case, you can either install and configure a Storybook addon (e.g., [`@storybook/addon-styling`](https://github.com/storybookjs/addon-styling#storybookaddon-styling)), or customize [Storybook's webpack configuration](../builders/webpack.md#extending-storybooks-webpack-config) and include the appropriate loader.
+import '../src/styles/global.css';
+```
+
+If your component files import their CSS files, this will work too. The noticeable exception to this is if you're using CSS processor tools like Sass or Postcss.
 
 <FeatureSnippets paths={['configure/css-troubleshooting/angular.mdx']} />
 
-To use your CSS in all stories, you import it in [`.storybook/preview.js`](./overview.md#configure-story-rendering)
+## CSS processors
+
+If you're using Vite as your builder, you're covered! Vite supports Sass and PostCSS out-of-the-box 🎉
+
+However, if you're using Webpack and want to use Sass and PostCss, you'll need some extra configuration. We recommend installing [`@storybook/addon-styling`](https://github.com/storybookjs/addon-styling#storybookaddon-styling) to help you configure these tools. Or if you'd prefer, you can customize [Storybook's webpack configuration yourself](../builders/webpack.md#extending-storybooks-webpack-config) to include the appropriate loader(s).
```

Comment:
if you could adjust the Webpack link to be `../builders/webpack.md#override-the-default-configuration` as that portion of the documentation was updated as part of the changes for Storybook 7.0, and with that it will prevent a 404.